app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -110,7 +110,7 @@ public void onClick(View view) {
     }
 
     private void populateMainGroup(Menu menu) {
-        menu.add(GROUP_ID, DRAWER_HOME, Menu.NONE, R.string.home_gdg).setIcon(R.drawable.ic_drawer_home_gdg);
```

Comment:
Thanks for the great pr :) Finally we have the new icons. 

I was thinking I if this old png is used anywhere else or not. If not we should remove it. 

Also do you think that this vector will work on older devices? Would be good to provide Screenshots in the description of the pr